cryptography/src/commonJvmAndroid/kotlin/com.wire.kalium.cryptography/MLSClientImpl.kt

```diff
@@ -326,14 +326,14 @@ class MLSClientImpl(
             return clientId?.let {
                 WireIdentity(
                     CryptoQualifiedClientId.fromEncodedString(value.clientId)!!,
-                    value.handle,
-                    value.displayName,
-                    value.domain,
-                    value.certificate,
+                    value.x509Identity?.handle ?: "",
```

Comment:
Question: since the inner object is nullable. Imo we should return null instead of hardcoded values?